src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportHelperTest.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import org.junit.Test;
+
+public class ReportHelperTest {
+
+    @Test
+    public void shouldReturnNullForInputWithLengthLessThanTenChars() {
+        assertNull(ReportHelper.getFormattedLocalDate("2021-3-2"));
+    }
+
+    @Test
+    public void shouldReturnNullForUnParseableInput() {
+        // When un-parseable input string is supplied, an exception is thrown and handled.
+        // After logging an error message, null is returned.
+        assertNull(ReportHelper.getFormattedLocalDate("2021-06-25BAD 21:12:12"));
+    }
+
+    @Test
+    public void shouldReturnLocalDateForValidInput() {
```

Comment:
Using parameterized tests would be a better approach than adding multiple tests to one method.
I recommend updating your unit test to use JUnit 5 rather than JUnit 4. Then take a look at something like this https://www.baeldung.com/parameterized-tests-junit-5

Reply:
Point taken.

Reply:
There are also examples of parameterised tests in the test classes EventValidationServiceTest and HearingToJudgmentsReportTest if you wanted an example with a codebase you are already familiar with